integrationtests/foldersync/foldersync_test.go

```diff
@@ -0,0 +1,385 @@
+package foldersync
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	logging "github.com/ipfs/go-log"
+	ma "github.com/multiformats/go-multiaddr"
+	core "github.com/textileio/go-threads/core/db"
+	"github.com/textileio/go-threads/core/thread"
+	"github.com/textileio/go-threads/db"
+	"github.com/textileio/go-threads/util"
+)
+
+func TestMain(m *testing.M) {
+	logging.SetLogLevel("foldersync", "info")
+	// logging.SetLogLevel("store", "debug")
+	// logging.SetLogLevel("threads", "debug")
+	// logging.SetLogLevel("threadstore", "debug")
+	os.Exit(m.Run())
+}
+
+func TestSimple(t *testing.T) {
```

Comment:
Just to eliminate the complexity of the bootstrap test below, this test is super simple. Creates multiple DBs sharing a thread, adds data to them and checks for data synchronization